src/loader/multi_loader.rs

```diff
@@ -0,0 +1,70 @@
+use crate::Loader;
+use fluent_bundle::FluentValue;
+use std::borrow::Cow;
+use std::collections::HashMap;
+
+pub use unic_langid::LanguageIdentifier;
+
+/// A loader comprised of other loaders.
+///
+/// This loader allows for loaders with multiple sources to be used
+/// from a single one, instead of a multiple of them.
```

Comment:
This could use more documentation. I would include more information on *why* you would want to use multiple sources. It also needs a code example of constructing it. It should also include information about what is the behaviour when there are multiple sources, which source is chosen first?

Reply:
Are you satisfied with the docs now?